buildenv/jenkins/jobs/infrastructure/copyrightCheck

```diff
@@ -64,17 +64,34 @@ timeout(time: 6, unit: 'HOURS') {
                         sh 'exit 1'
```

Comment:
I know this is not the original scope of this issue, but can you create a second commit in this PR? The above `if/else` statement can be removed and directly set `REGEX` based on the `eclipse` case. We don't use this script for the ibmruntimes ext repos anymore.